packages/minuteman/extra/dcos-minuteman.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
+
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=5
+WorkingDirectory=/opt/mesosphere/active/minuteman/minuteman
+EnvironmentFile=/opt/mesosphere/etc/minuteman-erl.env
+EnvironmentFile=/opt/mesosphere/etc/check_time.env
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/minuteman.env
+EnvironmentFile=-/run/dcos/etc/minuteman_auth.env
+ExecStartPre=/opt/mesosphere/bin/check-time
+ExecStartPre=/usr/bin/env modprobe ip_vs_wlc
```

Comment:
Previously the minuteman code called iptables. That in turn caused it to automatically load the module inside of the iptables/ipset code (since it loads missing modules by itself).

This version of minuteman does not shell out to iptables/ipset anymore but still requires the same modules so we need to load it at some point.

I think it would be cleaner to have minuteman load the module itself and hard fail if it does not exist versus putting this  logic in the init code.